modules/standard/track/track_controller.py

```diff
@@ -0,0 +1,103 @@
+import time
+
+from core.buddy_service import BuddyService
+from core.command_param_types import Const, Character, Options
+from core.decorators import instance, command, event
+from core.chat_blob import ChatBlob
+
+
+@instance()
+class TrackController:
+    def inject(self, registry):
+        self.bot = registry.get_instance("bot")
+        self.db = registry.get_instance("db")
+        self.buddy_service = registry.get_instance("buddy_service")
+        self.util = registry.get_instance("util")
+        self.text = registry.get_instance("text")
+
+    def start(self):
+        self.db.exec("CREATE TABLE IF NOT EXISTS track (char_id INT NOT NULL PRIMARY KEY, added_by_char_id INT NOT NULL, created_at INT NOT NULL)")
+        self.db.exec("CREATE TABLE IF NOT EXISTS track_log (char_id INT NOT NULL, action VARCHAR(10) NOT NULL, created_at INT NOT NULL)")
+
+    @command(command="track", params=[], access_level="member",
+             description="Show the track list")
+    def track_cmd(self, request):
+        data = self.get_all_tracked_chars()
+        blob = ""
+        for row in data:
+            blob += self.text.make_tellcmd(row.name, "track %s" % row.name) + " - " + ("<green>Online</green>" if self.buddy_service.is_online(row.char_id) == True else "<red>Offline</red>") + "\n"
+
+        return ChatBlob("Track List", blob)
+
+    @command(command="track", params=[Const("add"), Character("char")], access_level="member",
+             description="Add a character to the track list")
+    def track_add_cmd(self, request, _, char):
+        if self.get_tracked_char(char.char_id):
+            return "already tracked message"
+        
+        self.add_tracked_char(char.char_id, request.sender.char_id)
+        return "%s is now being tracked." % char.name
+
+    @command(command="track", params=[Options(["rem", "remove"]), Character("char")], access_level="member",
+             description="Remove a character from the track list")
+    def track_remove_cmd(self, request, _, char):
+        if not self.get_tracked_char(char.char_id):
+            return "%s is already being tracked." % char.name
+
+        self.remove_tracked_char(char.char_id)
+        return "%s has been removed from the track list." % char.name
```

Comment:
we should standardize on either "is being tracked" or "added to the track list", either one is fine, but we should use the same terminology throughout